jujugui/static/gui/src/app/components/machine-view/test-machine-view.js

```diff
@@ -699,6 +699,7 @@ describe('MachineView', function() {
       '.machine-view__auto-place .button--inline-neutral');
     testUtils.Simulate.click(node);
     assert.equal(autoPlaceUnits.callCount, 1);
+    ReactDOM.unmountComponentAtNode(ReactDOM.findDOMNode(component).parentNode);
```

Comment:
This test needed to unmount components because it was causing cascading failures. These were caused by the `enhancewithclickoutside` not detaching its click event handlers.